tsconfig.base.json

```diff
@@ -14,12 +14,7 @@
     "skipLibCheck": true,
     "skipDefaultLibCheck": true,
     "baseUrl": ".",
-    "paths": {
-      "@findr/mui": ["packages/fnr-mui/src/index.ts"],
```

Comment:
These are needed, Nx uses this to calculate relationship between packages.

Reply:
Are you sure? The `paths` key is intended to be used for import aliases. I think the `references` key in each sub project is what you need.

Reply:
Here's a link to what I mean: https://www.typescriptlang.org/docs/handbook/project-references.html
Also, it looks like this could be a more maintainable solution: https://nx.dev/concepts/more-concepts/how-project-graph-is-built#how-the-project-graph-is-built

Reply:
https://nx.dev/getting-started/tutorials/integrated-repo-tutorial#local-linking-of-packages